model/flow/service_event.go

```diff
@@ -34,6 +24,27 @@ type ServiceEvent struct {
 	Event interface{}
 }
 
+// ServiceEventsList is a handy container to enable comparisons
+type ServiceEventsList []ServiceEvent
```

Comment:
```suggestion
// ServiceEventList is a handy container to enable comparisons
type ServiceEventList []ServiceEvent
```
To match our other `*List` types